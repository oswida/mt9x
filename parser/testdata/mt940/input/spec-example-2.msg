:20:123456
:25P:123-304958
CORPGB22
:28C:123/1
:60F:C090123USD395212311,71
:61:090123C50000000,NTRFNONREF//8951234
ORDER BK OF NYC WESTERN CASH RESERVE
:61:090126C5700000,NFEX036960//8954321
:61:090127C200000,NDIVNONREF//8846543
:86:DIVIDEND LORAL CORP
PREFERRED STOCK 1ST QUARTER 2009
:62F:C090123USD451112311,71
:64:C090123USD445212311,71
:65:C090126USD450912311,71
:65:C090127USD451112311,71
:86:PRIME RATE AS OF TODAY 11 PCT