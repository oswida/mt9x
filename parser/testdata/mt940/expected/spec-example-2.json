{
 "tag20": "123456",
 "tag25": {
  "account": "123-304958",
  "ident_code": "CORPGB22"
 },
 "tag28": {
  "stmt_number": "123",
  "seq_number": "1"
 },
 "tag60": {
  "dc_mark": "C",
  "date": "2009-01-23T00:00:00Z",
  "currency": "USD",
  "amount": "395212311.71"
 },
 "statements": [
  {
   "tag61": {
    "value_date": "2009-01-23T00:00:00Z",
    "dc_mark": "C",
    "amount": "50000000",
    "trx_ident": "NTRF",
    "owner_ref": "NONREF",
    "institution_ref": "8951234",
    "details": "ORDER BK OF NYC WESTERN CASH RESERVE"
   }
  },
  {
   "tag61": {
    "value_date": "2009-01-26T00:00:00Z",
    "dc_mark": "C",
    "amount": "5700000",
    "trx_ident": "NFEX",
    "owner_ref": "036960",
    "institution_ref": "8954321"
   }
  },
  {
   "tag61": {
    "value_date": "2009-01-27T00:00:00Z",
    "dc_mark": "C",
    "amount": "200000",
    "trx_ident": "NDIV",
    "owner_ref": "NONREF",
    "institution_ref": "8846543"
   },
   "tag86": [
    "DIVIDEND LORAL CORP",
    "PREFERRED STOCK 1ST QUARTER 2009"
   ]
  }
 ],
 "tag62": {
  "dc_mark": "C",
  "date": "2009-01-23T00:00:00Z",
  "currency": "USD",
  "amount": "451112311.71"
 },
 "tag64": {
  "dc_mark": "C",
  "date": "2009-01-23T00:00:00Z",
  "currency": "USD",
  "amount": "445212311.71"
 },
 "tag65": [
  {
   "dc_mark": "C",
   "date": "2009-01-26T00:00:00Z",
   "currency": "USD",
   "amount": "450912311.71"
  },
  {
   "dc_mark": "C",
   "date": "2009-01-27T00:00:00Z",
   "currency": "USD",
   "amount": "451112311.71"
  }
 ],
 "tag86": [
  "PRIME RATE AS OF TODAY 11 PCT"
 ]
}